technologist.gemspec

```diff
@@ -10,8 +10,9 @@ Gem::Specification.new do |spec|
   spec.email         = ["mail@koffeinfrei.org"]
 
   spec.summary       = %q{Detects the technologies used in a repository.}
-  spec.description   = %q{}
+  spec.description   = %q{Detects technologies in a repository by applying a set of rules and returning base and secondary frameworks.}
```

Comment:
Maybe "returning primary and" as the method is called `primary_frameworks` as well.